src/button-dropdown/category-elements/styles.scss

```diff
@@ -17,7 +17,8 @@
   border-inline-width: 0;
   font-weight: bold;
   display: flex;
-  justify-content: space-between;
+  align-items: center;
```

Comment:
This keeps the icon at the vertical center when the text wraps:

<img width="137" height="62" alt="Screenshot 2025-07-25 at 18 09 13" src="https://github.com/user-attachments/assets/324f6629-9584-44c3-aa04-4d887a892ebd" />

whereas in non-group items it currently stays aligned to the top:

<img width="172" height="100" alt="Screenshot 2025-07-25 at 18 14 58" src="https://github.com/user-attachments/assets/db27127a-1c5b-4efc-8431-27f08c854c07" />